cpp/ql/test/library-tests/dataflow/fields/ir-flow.ql

```diff
@@ -41,6 +37,28 @@ class Conf extends DataFlow::Configuration {
   }
 }
 
-from DataFlow::PathNode src, DataFlow::PathNode sink, Conf conf
-where conf.hasFlowPath(src, sink)
-select sink, src, sink, sink + " flows from $@", src, src.toString()
+class IRFieldFlowTest extends InlineExpectationsTest {
+  IRFieldFlowTest() { this = "IRFieldFlowTest" }
+
+  override string getARelevantTag() { result = "ir" }
+
+  override predicate hasActualResult(Location location, string element, string tag, string value) {
+    exists(Node source, Node sink, Conf conf, int n |
+      tag = "ir" and
+      conf.hasFlow(source, sink) and
+      n = strictcount(Node otherSource | conf.hasFlow(otherSource, sink)) and
+      (
+        n = 1 and value = "flow"
```

Comment:
The test framework lets you use an empty string for `value`, and if you do that, you can omit the `=flow` part of all of your expectations. It's up to you whether you think that's more or less readable.

Reply:
I've removed the "=flow". It's definitely more readable.